src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -208,6 +208,10 @@ public function load(array $configs, ContainerBuilder $container)
             $this->registerPropertyInfoConfiguration($config['property_info'], $container, $loader);
         }
 
+        if ($this->isConfigEnabled($container, $config['links'])) {
```

Comment:
I think we should test for the Link component to be present first and throw a meaningful exception if not present.